lib/Fhp/Segment/DSE/ParameterTerminierteSEPAEinzellastschriftEinreichenV2.php

```diff
@@ -2,13 +2,25 @@
 
 namespace Fhp\Segment\DSE;
 
-use Fhp\Segment\DME\ParameterTerminierteSEPALastschriftEinreichenV2;
+use Fhp\Segment\BaseDeg;
 
-class ParameterTerminierteSEPAEinzellastschriftEinreichenV2 extends ParameterTerminierteSEPALastschriftEinreichenV2
+class ParameterTerminierteSEPALastschriftEinreichenV2 extends BaseDeg implements SEPADirectDebitMinimalLeadTimeProvider
```

Comment:
My IDE (PhpStorm) rightfully complains: "Class name doesn't match the containing file name" because there's "Einzel" missing.

If your IDE isn't able to point this out, you may want to check out one that can. Since PHP doesn't have a compiler, it saves a lot of pain to have some other form of static checking that points out bugs before one runs the code for the first time.